experience.html

```diff
@@ -0,0 +1,50 @@
+<html>
+	<head>
+		<head>
+		 <meta charset="UTF-8">
+		<title>Robert Plant the greatest</title>
+		<link rel="stylesheet" href="css/style.css" media="screen" type="text/css" />
+	</head>
+	</head>
+	<body>
+		<h2>the Robert Plant Experience</h2>
+		<nav>
+			<a href="index.html">Home</a> |
+ 			<a href="experience.html">Robert Plant -The Experience</a> |
+ 			<a href="https://vimeo.com/search?q=led+zeppelin"  target="_blank" >Led Zeppelin on vimeo</a>
+		</nav>
+		<hr class="style1 ">
+		<h1>Robert Plant</h1>
+		<h4>The Experience</h4>
+		
+
+		<p>
+			Life and career[edit]
+Early life and musical beginnings[edit]
+Robert Anthony Plant was born in the Black Country town of West Bromwich, Staffordshire to Robert C. Plant, a qualified civil engineer who worked in the Royal Air Force during World War II,[6] and Annie Celia Plant (née Cain),[7] a Romanichal woman.[8] He grew up in Kidderminster, Worcestershire. Plant gained an interest in singing and rock and roll music at an early age.
+
+		</p>
+
+		<p>
+			Life and career[edit]
+Early life and musical beginnings[edit]
+Robert Anthony Plant was born in the Black Country town of West Bromwich, Staffordshire to Robert C. Plant, a qualified civil engineer who worked in the Royal Air Force during World War II,[6] and Annie Celia Plant (née Cain),[7] a Romanichal woman.[8] He grew up in Kidderminster, Worcestershire. Plant gained an interest in singing and rock and roll music at an early age.
+		</p>
+		<p>
+			“	I suppose I was quite interested in my stamp collection and Romano-British history. I was a little grammar school boy and I could hear this kind of calling through the airwaves.[10]	”
+He abandoned training as a chartered accountant after only two weeks to attend college in an effort to gain more GCE passes (General Certificates of Education) and to become part of the English Midlands blues scene.[11][12] "I left home at 16", he said, "and I started my real education musically, moving from group to group, furthering my knowledge of the blues and of other music which had weight and was worth listening to".[13]
+		</p>
+
+		<p>
+Plant's early blues influences included Johnson, Bukka White, Skip James, Jerry Miller, and Sleepy John Estes. Plant had various jobs while pursuing his music career, one of which was working for the major British construction company Wimpey in Birmingham in 1967 laying tarmac on roads. He also worked at Woolworth's in Halesowen town for a short period of time. He cut three obscure singles on CBS Records[14] and sang with a variety of bands, including the Crawling King Snakes, which brought him into contact with drummer John Bonham. They both went on to play in the Band of Joy, merging blues with newer psychedelic trends.
+
+		</p>
+
+		<hr class="style1 ">
+		<h4>Thank you for visiting the fanclub at Robert Plant</h4>
+
```

Comment:
:hammer: Best practices would be to use a _single_ empty line to separate sections of your code. I would remove unnecessary empty lines like here at the end.